src/Examples.Console/Program.cs

```diff
@@ -46,7 +44,7 @@ internal class DynatraceOptions
 		[Option('t', "token", HelpText = "Dynatrace API authentication token with the 'metrics.ingest' permission.", Required = false)]
 		public string ApiToken { get; set; }
 
-		[Option('n', "noDynatraceMetadataEnrichment", HelpText = "Disable automatic label enrichment via Dynatrace metadata.", Required = false)]
-		public bool DisableDynatraceMetadataEnrichment { get; set; } = false;
+		[Option('n', "enableDynatraceMetadataEnrichment", Default = true, HelpText = "Enables automatic label enrichment via Dynatrace metadata.", Required = false)]
```

Comment:
should the short flag still be `-n`? :thinking:

Reply:
Hm, i agree `n` might not be intuitive.. maybe `m` for metadata?

Reply:
`m` it is.